### PR TITLE
About: Implement partners grid

### PR DESCRIPTION
**Pull request by iandenty**
**NOTE:** Currently no partner records, therefore partners not rendered.

Closes https://www.pivotaltracker.com/story/show/154403530

### CODE REVIEW

File: components/ui/partner-card.js [hunk 1]
```diff
@@ -1,41 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-class PartnerCard extends React.Component {
-  static propTypes = {
```
Comment (mbarrenechea) [inline components/ui/partner-card.js]: Why did you remove these static props and move them to the bottom? I know that every other component has them at the bottom. But I think that is more readable and more usable to have them here. Whenever you enter a component you will know which props are allowed and the type of them

File: components/ui/partner-card.js [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
-          <p>
-            {description}
-          </p>
-        </div>
+  return (
+    <div className="c-partner-card">
+      <div className="partner-card-logo-container">
+        <a rel="noopener noreferrer" target="_blank" href={website}>
+          <h3 className="partner-card-title">{name}</h3>
+          <picture>
```
Comment (mbarrenechea) [inline components/ui/partner-card.js]: image

File: components/ui/partner-card.js (hunk 1 above)
Comment (iandenty) [reply components/ui/partner-card.js]: Happy to move `propTypes` back to a static property. I agree, the component is more understandable with the `propTypes` up front. I was aligning to the other components I had encountered - consistency across components important to readability and usability too.

I'll move back and ensure new components have static property for `propTypes`.

Comment (mbarrenechea) [reply components/ui/partner-card.js]: Yes, let's do that whenever we can

File: components/ui/partner-card.js (hunk 2 above)
Comment (iandenty) [reply components/ui/partner-card.js]: I was thinking that the `<picture>` element gracefully degrades to the `` element in unsupported browsers 🤔.

Comment (mbarrenechea) [reply components/ui/partner-card.js]: Check it and sent the confirmation. In that case, BRAVO

Comment (iandenty) [reply components/ui/partner-card.js]: This was old w3c draft spec, can't say anything explicitly in latest, would need to confirm by testing on an older browser:
image

From https://developers.google.com/web/ilt/pwa/lab-responsive-images: 
image

Comment (mbarrenechea) [reply components/ui/partner-card.js]: 👍

**Comment by mbarrenechea** [comment]
Ok, so if you change back the static props to its site I will merge, I promiss...

**Comment by iandenty** [comment]
One more thing 😬

With this `PartnerCard` component:

I had this warning with the `class` component, which I think was another reason for me moving the propTypes:

`[eslint] Component should be written as a pure function (react/prefer-stateless-function)`

If changing to functional component, don't have access to `static`?

**Comment by mbarrenechea** [comment]
If you want to avoid this eslint comment, you can define a static defaultProps and it will be gone

**Comment by iandenty** [comment]
@mbarrenechea updated `parnter-card` component structure.
